src/main/java/unrc/dose/Comment.java

```diff
@@ -8,10 +13,88 @@
 * description     :varchar(300)    not null
 * challenge_id    :integer     not null
 * user_id         :integer    not null
-**/
-package unrc.dose;
+* comment_id      :integer
+*/
+public class Comment extends Model {
 
-import org.javalite.activejdbc.Model;
+  /**
+  *Create a new comment.
+  *@param title this is the comment's title
+  *@param description this the comment's body
+  *@param challengeId this is the challenge's id which is commented
+  *@param userId this is the user's id who commented
+  *@return the comment created
+  **/
+  public static Comment createComment(final String title,
+  final String description, final int challengeId, final int userId) {
+    Comment c = new Comment();
+    c.set("title", title);
+    c.set("description", description);
+    c.set("challenge_id", challengeId);
+    c.set("user_id", userId);
+    c.saveIt();
+    return c;
+  }
+
+  /**
+  *isResponse.
+  *@param commentId this is comments or response to evaluate
+  *@return return if the selected comment_id si a comment or a response.
+  **/
+  public static boolean isResponse(final int commentId) {
+    Comment c = Comment.findById(commentId);
+    return ((c.getInteger("comment_id")) != (null));
+  }
+
+  /**
+  *Create a new Response.
+  *@param description this the response's body
+  *@param userId this is the user's id who responded to a comment
+  *@param commentId this is the id to the comment it references
+  *@return the created response
+  *@throws NullPointerException when doesn't exits the comment which this id
+  **/
+  public static Comment createResponse(final String description,
+  final int userId, final int commentId) throws NullPointerException {
```

Comment:
A better format for this instance where there are too many arguments could be:

```java
public static Comment createResponse(
    final String description,
    final int userId
    final int challengeId)
{
```

`NullPointerException` is a type of unchecked exception, and as such they don't need to be defined in a method's `throws` declaration.

The order of this method's `userId` and `challengeId` arguments is not the same as in the `createComment` method.